.github/workflows/go.yml

```diff
@@ -54,7 +54,7 @@ jobs:
       - name: Golangci-lint
         uses: golangci/golangci-lint-action@v3
         with:
-          version: v1.52.2
+          version: latest
```

Comment:
Can you pin to a specific version please? We've had pain in the past where this is suddenly incompatible and blocking all PRs

Reply:
Yes, will update.